docs/modules/user/pages/quickstart.adoc

```diff
@@ -364,6 +364,255 @@ docker run \
+lxc.mount.entry: /dev/ttyUSB1       dev/ttyUSB1       none bind,optional,create=file
+lxc.mount.entry: /dev/ttyACM0       dev/ttyACM0       none bind,optional,create=file
+lxc.mount.entry: /dev/ttyACM1       dev/ttyACM1       none bind,optional,create=file
+```
+
+add these to the bottom of the file:
+
+```
+dev0: /dev/uinput
+dev1: /dev/uhid
+dev2: /dev/nvidia0
+dev3: /dev/nvidiactl
+dev4: /dev/nvidia-modeset
+dev5: /dev/nvidia-uvm
+dev6: /dev/nvidia-uvm-tools
+dev7: /dev/nvidia-caps/nvidia-cap1
+dev8: /dev/nvidia-caps/nvidia-cap2
+lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
+lxc.mount.entry: /dev/input dev/input none bind,optional,create=dir
+lxc.mount.entry: /run/udev mnt/udev none bind,optional,create=dir
+```
+
+save the file, exit, and restart your LXC.
+
+From now on, we will work inside the LXC.
+
+First, let's build an additional docker image that will contain the Nvidia driver files:
+
+[source,bash]
+....
+curl https://raw.githubusercontent.com/games-on-whales/gow/master/images/nvidia-driver/Dockerfile | docker build -t gow/nvidia-driver:latest -f - --build-arg NV_VERSION=$(cat /sys/module/nvidia/version) .
+....
+
+This will create `gow/nvidia-driver:latest` locally.
+
+Unfortunately, docker doesn't seem to support directly mounting images, but you can https://docs.docker.com/storage/volumes/#populate-a-volume-using-a-container[pre-polulate volumes] by running:
+
+[source,bash]
+....
+docker create --rm --mount source=nvidia-driver-vol,destination=/usr/nvidia gow/nvidia-driver:latest sh
+....
+
+It will create a Docker container, populate `nvidia-driver-vol` with Nvidia driver if it wasn't already done and remove the container.
+
+Check volume exists with:
+
+[source,bash]
+....
+docker volume ls | grep nvidia-driver
+
+local     nvidia-driver-vol
+....
+
+By now we have all the setup in place, let's create our docker-compose.yml:
+
+```
+services:
+  wolf:
+    image: ghcr.io/games-on-whales/wolf:stable
+    environment:
+      - XDG_RUNTIME_DIR=/tmp/sockets
+      - NVIDIA_DRIVER_VOLUME_NAME=nvidia-driver-vol
+      - HOST_APPS_STATE_FOLDER=/etc/wolf
+      # This one below is only for multi GPU setup
+      #- WOLF_RENDER_NODE=/dev/dri/renderD129 
+    volumes:
+      - /etc/wolf/:/etc/wolf:rw
+      - /tmp/sockets:/tmp/sockets:rw
+      - /var/run/docker.sock:/var/run/docker.sock:rw
+      - /dev/:/dev/:rw
+      # This one below uses the /mnt/udev path we created in the LXC config file
+      - /mnt/udev:/run/udev:rw 
+      - nvidia-driver-vol:/usr/nvidia:rw
+    devices:
+      - /dev/dri
+      - /dev/uhid
+      - /dev/uinput
+      - /dev/nvidia-uvm
+      - /dev/nvidia-uvm-tools
+      - /dev/nvidia-caps/nvidia-cap1
+      - /dev/nvidia-caps/nvidia-cap2
+      - /dev/nvidiactl
+      - /dev/nvidia0
+      - /dev/nvidia-modeset
+    device_cgroup_rules:
+      - 'c 13:* rmw'
+    network_mode: host
+    restart: always
+
+volumes:
+  nvidia-driver-vol:
+    external: true
+```
```

Comment:
This is the same as the normal guide, right? Can't we just say that from this point on you can follow the guide on the other tabs?

Reply:
pretty much, I'll do that and just add a couple remarks